lib/modules/manager/mix/artifacts.ts

```diff
@@ -26,15 +28,35 @@ export async function updateArtifacts({
   config,
 }: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
   logger.debug(`mix.getArtifacts(${packageFileName})`);
-  if (updatedDeps.length < 1) {
-    logger.debug('No updated mix deps - returning null');
+  const isLockFileMaintenance = config.updateType === 'lockFileMaintenance';
+
+  if (is.emptyArray(updatedDeps) && !isLockFileMaintenance) {
+    logger.debug('No updated mix deps');
+    return null;
+  }
+
+  let lockFileName = getSiblingFileName(packageFileName, 'mix.lock');
+  let isUmbrella = false;
+
+  let existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+  if (!existingLockFileContent) {
+    lockFileName =
+      (await findLocalSiblingOrParent(packageFileName, 'mix.lock')) ??
+      'mix.lock';
+    existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+    isUmbrella = !!existingLockFileContent;
+  }
+
+  if (isLockFileMaintenance && isUmbrella) {
+    logger.debug('Cannot use lockFileMaintenance in an umbrella project');
```

Comment:
we should log a url to a description why it's not possible.
```suggestion
    logger.debug('Cannot use lockFileMaintenance in an umbrella project, see https://docs.renovatebot.com/modules/manager/mix/#lockFileMaintenance');
```